internal/controller/dnsrecord_controller.go

```diff
@@ -46,18 +44,15 @@ import (
 
 const (
 	DNSRecordFinalizer        = "kuadrant.io/dns-record"
-	WriteCounterLimit         = 20
 	validationRequeueVariance = 0.5
-	DefaultValidationDuration = time.Second * 5
 )
 
 var (
-	defaultRequeueTime    time.Duration
-	validationRequeueTime time.Duration
-	noRequeueDuration     = time.Duration(0)
-	validFor              time.Duration
-	reconcileStart                    = metav1.Time{}
-	Clock                 clock.Clock = clock.RealClock{}
+	defaultRequeueTime       time.Duration
+	defaultValidationRequeue time.Duration
+	validationRequeueTime    time.Duration
```

Comment:
This var is always the randomized requeue, right? I think the name makes this confusing, it would be better with something like: `randomizedRequeueTime`. Though I'm not convinced that it's required at this scope... why not just calculate it at the point it is required?

Reply:
It is used a few times across the code and if we will calculate it every time we need that we will end up with different values in one instance (right now will not cause any troubles, but might in future) and it seems cleaner to me to randomize once and then just use the var. But I do have a problem with naming :laughing: